packages/connect-react/src/utils/component.ts

```
@@ -0,0 +1,183 @@
+  } = opts
+  const value = valueFromOption(valueOpt)
+
+  if (!prop.default && value == null || typeof value === "undefined") return [
+    "required",
+  ]
+
+  const _value: number = typeof value === "number"
+    ? value
+    : parseInt(String(value))
+
+  if (_value !== _value) return [
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)